packages/peewee/test_peewee.py

```diff
@@ -0,0 +1,40 @@
+import os
```

Comment:
You'll need to:
1. Import `run_in_pyodide` at top level
2. Move other imports into the test function.

Globals in the file are not visible inside the function.